### DISCUSSION TITLE
Diff snapshots versions

### DISCUSSION THREAD

**Discussion by dwaddington** [firecracker-microvm/firecracker#2808] [Q&A] [answered]
Hi,
I using the main branch of firecracker and since updating from v0.25 I am unable to perform incremental snap shots.
The error is: {"fault_message":"The virtio devices use a features that is incompatible with older versions of Firecracker: notification suppression"}HTTP/1.1 204 

Can somebody explain what this means?

Thanks
Daniel

**Comment by luminitavoicu** [comment, marked as answer]
Hi @dwaddington,

The main branch includes notification suppression for the virtio devices which will be released in Firecracker v1.0. The notification suppression feature is incompatible with older Firecracker versions, which means that creating a snapshot with target version lower than 1.0.0 (which I assume you are doing) will fail.

In order for you to be able to create snapshots with version 0.25.0 or lower, you could switch to using Firecracker v0.25.2. Otherwise, in order for the snapshot operation to work on the main branch, you should specify "1.0.0" as target version.
